README.md

```diff
@@ -1 +1,95 @@
-# deeplearnjs-legacy-loader
+# deeplearn.js legacy loader
+
+This repo provides the legacy loader for porting a TensorFlow model to deeplearn.js.
```

Comment:
Can you just again say this is deprecated and this is not the recommended way?

Reply:
done.